llumnix/manager.py

```diff
@@ -137,37 +134,29 @@ def __init__(self,
 
     async def generate(self, request_id: RequestIDType, server_info: ServerInfo, *args, **kwargs) -> None:
         while self.num_instances == 0:
-            logger.warning("No instance available now, sleep {}s, "
-                           "and regenerate request {}.".format(NO_INSTANCE_RETRY_GENERATE_INTERVAL, request_id))
+            logger.warning("No instance available now, sleep {}s, and regenerate request {}.".format(
+                NO_INSTANCE_RETRY_GENERATE_INTERVAL, request_id))
             await asyncio.sleep(NO_INSTANCE_RETRY_GENERATE_INTERVAL)
-        prefill_instance_id, request_expected_steps = self.global_scheduler.dispatch(InstanceType.PREFILL)
-        if self.manager_args.enable_engine_pd_disagg:
-            # Only used in bladellm now
-            decode_instance_id, _ = self.global_scheduler.dispatch(InstanceType.DECODE)
-            kwargs["decode_instance_id"] = self.instance_id_2_engine_disagg_inst_id.get(
-                decode_instance_id, None
-            )
+
+        instance_id, request_expected_steps, addition_dispatch_info = self.global_scheduler.dispatch(request_id)
```

Comment:
i dont like this addition_dispatch_info design, too implicit, i prefer the original codes.